backend/src/litigation_api/models/api.py

```
@@ -1,7 +1,7 @@
-from pydantic import BaseModel, Field
+from pydantic import BaseModel
 from typing import List, Generic, TypeVar, Optional
```

Ruff (0.15.11)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)